File: week-1/excel-sheet-column-number/excel-sheet-column-number.py
class Solution:
    def titleToNumber(self, s: str) -> int:
        res = 0
        placeValue = len(s)-1
        for i in s:
            res += int(ord(i) - 64) * (26**placeValue)
            placeValue -= 1
        
        return res

File: week-1/excel-sheet-column-number/excel-sheet-column-number.js
/**
 * @param {string} s
 * @return {number}
 */
var titleToNumber = function(s) {
    let res = 0;
    let placeValue = s.length - 1;
    s.split('').forEach(i => {
        res += (i.charCodeAt() - 64) * (26**placeValue);
        placeValue--;
    });
    return res;
};
